src/components/exercises/App1/redux/slices/userSlice.js

```diff
@@ -39,6 +57,7 @@ export const userSlice = createSlice({
     logInError: false,
     username: "",
     password: "",
+    submitedData: "",
```

Comment:
Why this new property?

Reply:
to access the data and show them as a message when profile data saved